app/javascript/declarations.d.ts

```diff
@@ -28,3 +28,18 @@ declare module 'reconnecting-websocket' {
   const ReconnectingWebsocket: ReconnectingWebsocket
   export = ReconnectingWebsocket
 }
+
```

Comment:
I needed to declare my own type declarations for the vim plugin. There is no defaults.